go/database/mpt/config.go

```diff
@@ -84,6 +86,17 @@ func GetConfigByName(name string) (MptConfig, bool) {
 	return MptConfig{}, false
 }
 
+// GetEncoders returns the value encoders for the given MPT configuration.
+func (c MptConfig) GetEncoders() (
+	stock.ValueEncoder[AccountNode],
+	stock.ValueEncoder[BranchNode],
+	stock.ValueEncoder[ExtensionNode],
+	stock.ValueEncoder[ValueNode],
+) {
+
+	return getEncoder(c)
```

Comment:
nit: In this case I think we should replace `getEncoder(c)` with `c.GetEncoders()` everywhere ... wdyt?

Reply:
It could be, yes, I will update in a follow up